src/main/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtil.java

```diff
@@ -95,18 +95,15 @@ private JavadocUtil() {
      */
     public static JavadocTags getJavadocTags(TextBlock textBlock,
             JavadocTagType tagType) {
-        final boolean getBlockTags = tagType == JavadocTagType.ALL
-                                         || tagType == JavadocTagType.BLOCK;
-        final boolean getInlineTags = tagType == JavadocTagType.ALL
-                                          || tagType == JavadocTagType.INLINE;
-
         final List<TagInfo> tags = new ArrayList<>();
-
-        if (getBlockTags) {
+        final boolean isBlockTags = tagType == JavadocTagType.ALL
+                                        || tagType == JavadocTagType.BLOCK;
+        if (isBlockTags) {
```

Comment:
Not really sure I prefer this inspection. It moves it closer to it's usage, but its the still the same execution. It is not like the move inside if check, where it could prevent it being executed in certain cases.